interface-definitions/interfaces-openvpn.xml.in

```diff
@@ -635,6 +635,87 @@
                 </properties>
                 <defaultValue>net30</defaultValue>
               </leafNode>
+              <node name="2fa">
+                <properties>
+                  <help>2-factor authentication</help>
```

Comment:
I think we should spell it "Two-factor authentication" when it's used as a word rather than as an abbreviation (in the abbreviated form 2FA is fine).